examples/cluster/plot_inductive_clustering.py

```diff
@@ -0,0 +1,99 @@
+"""
+==============================================
+Inductive Clustering
+==============================================
+
+Clustering is expensive, especially when our dataset contains millions of
+datapoints. Recomputing the clusters everytime we receive some new data
+is thus in many cases, intractable. With more data, there is also the
+possibility of degrading the previous clustering.
```

Comment:
I think there is less of an issue with degrading, than with identifying the clusters across two clusterings. 

For that reason and others, this kind of technique is interesting regardless of the size of the dataset. An algorithm like agglomerative clustering or dbscan makes no hypothesis about how to divide the data in terms of features. Learning a classifier may also help us make inferences about the nature of the clustering. For this reason, I think we should aim to plot the decision boundary in the plot below

Reply:
>For that reason and others, this kind of technique is interesting regardless of the size of the dataset.  

@jnothman I agree. I kept the docstring from the original PR.
 
>An algorithm like agglomerative clustering or dbscan makes no hypothesis about how to divide the data in terms of features. Learning a classifier may also help us make inferences about the nature of the clustering. For this reason, I think we should aim to plot the decision boundary in the plot below

I agree again. I have plotted  the decision regions in the third plot. What do you think?

Reply:
The decision regions are helpful. Please also update the description to better match real use cases.